tests/plugins/tableselection/manual/scrollmouseover.md

```diff
@@ -6,8 +6,7 @@
 
 1. Open developer console.
 2. Focus on editor.
-3. Scroll down/up.
-4. Mouse over scrollbar.
+3. Use mouse to scroll editor and point the cursor over the scrollbar.
```

Comment:
This sentence is not clear enough. Enough would be some thing "Use mouse to scroll down the editor." - and simple as that. It implies that there will be cursor over the scroll bar.